cpp/tests/io/json/json_writer.cpp

```diff
@@ -611,4 +611,39 @@ TEST_P(JsonCompressedWriterTest, Unicode)
   run_test(out_options, expected);
 }
 
+TEST_P(JsonCompressedWriterTest, UnicodeUnescaped)
+{
+  cudf::test::strings_column_wrapper col1{"\"\\/\b\f\n\r\t", "ராபிட்ஸ்", "$€𐐷𤭢", "C𝞵𝓓𝒻"};
+  cudf::test::strings_column_wrapper col2{"CႮ≪ㇳ䍏凹沦王辿龸ꁗ믜스폶ﴠ",
+                                          "𐀀𑿪𒐦𓃰𔙆 𖦆𗿿𘳕𚿾[↳] 𜽆𝓚𞤁🄰",
+                                          "𠘨𡥌𢗉𣇊𤊩𥅽𦉱𧴱𨁲𩁹𪐢𫇭𬬭𭺷𮊦屮",
+                                          "𰾑𱔈𲍉"};
+  cudf::test::fixed_width_column_wrapper<int16_t> col3{{1, 2, 3, 4},
+                                                       cudf::test::iterators::nulls_at({0, 2})};
+  cudf::table_view tbl_view{{col1, col2, col3}};
+  cudf::io::table_metadata mt{{{"col1"}, {"col2"}, {"int16"}}};
+
+  std::vector<char> out_buffer;
+  auto destination = cudf::io::sink_info(&out_buffer);
+  auto out_options = cudf::io::json_writer_options_builder(destination, tbl_view)
+                       .include_nulls(true)
+                       .metadata(mt)
+                       .lines(true)
+                       .na_rep("null")
+                       .utf8_escaped(false)
+                       .build();
+
+  cudf::io::write_json(out_options, cudf::test::get_default_stream());
+
+  std::string const expected =
+    u8R"({"col1":"\"\\\/\b\f\n\r\t","col2":"CႮ≪ㇳ䍏凹沦王辿龸ꁗ믜스폶ﴠ","int16":null}
```

Comment:
What is `u8R` ?

Reply:
https://en.cppreference.com/w/cpp/language/string_literal
I see 2 basic versions. Rest all looks same.